spec/requests/api/chargebacks_assignement_spec.rb

```diff
@@ -0,0 +1,72 @@
+RSpec.describe "chargebacks API" do
+  it "can fetch the list of all tags of one chargeback" do
+    # Build Chargeback tag
+    chargeback = FactoryGirl.create(:chargeback_rate)
+    tenant     = FactoryGirl.create(:tenant)
+    ChargebackRate.set_assignments chargeback.rate_type, [{:cb_rate => chargeback, :object => tenant}]
```

Comment:
I think create a ChargebackRate assigned with a tenant directly could be cool ^^